frontend/src/components/ExistingSamples.vue

```diff
@@ -166,6 +175,9 @@
             <span class="remove-pwg-link" v-if="entry.interested_pwgs.includes(selectedPWG)" @click="removePWGFromEntriesAction(selectedPWG, [entry])">Remove {{selectedPWG}}</span>
           </template>
         </td>
+        <td style="max-width: 100px; min-height: 50px; min-width: 50px; overflow: auto">
+          {{entry.tags}}
```

Comment:
Would it be possible to add the "Add tag" and "Remove tag" buttons like in interested PWGs cell?